package.json

```diff
@@ -16,17 +16,29 @@
       "transform-flow-strip-types"
     ]
   },
+  "eslintConfig": {
+    "extends": "ello",
+    "globals": {
+      "AndroidInterface": false,
+      "ENV": false
```

Comment:
I would image we don't want either of these globals in brains. `AndroidInterface` should probably only live in webapp and `ENV` shouldn't be accessed via a 3rd party package.